docs/content/preview/explore/change-data-capture/using-logical-replication/_index.md

```diff
@@ -112,3 +112,5 @@ For reference documentation, see [YugabyteDB Connector](./yugabytedb-connector/)
 - Support for transaction savepoints is tracked in issue [10936](https://github.com/yugabyte/yugabyte-db/issues/10936).
 
 - Support for enabling CDC on Read Replicas is tracked in issue [11116](https://github.com/yugabyte/yugabyte-db/issues/11116).
+
+- Support for tablet splitting with logical replication is disabled from YB version 2024.1.4 & 2024.2.1. It is tracked in issue [24918](https://github.com/yugabyte/yugabyte-db/issues/24918).
```

Comment:
@siddharth2411 has a change for this already i.e. https://github.com/yugabyte/yugabyte-db/pull/25228

You can either remove this from your diff or @siddharth2411 can close his PR.

Reply:
Had discussed this offline with @siddharth2411. He will abandon his PR.